### DISCUSSION TITLE
Western notehead color

### DISCUSSION THREAD

**Discussion by LCagmac** [DigiScore/neoscore#54] [Q&A] [answered]
Hi,

Is there a way to change the notehead color for the western notation?

Thanks!

**Comment by ajyoon** [comment, marked as answer]
Hey there! It's a bit clumsy and under-documented, but it's certainly possible. I've just added [a new example which shows how to color noteheads and other component objects in `Chordrest`s](https://github.com/DigiScore/neoscore/blob/main/examples/chordrest_coloring.py). It also documents some gotchas with the approach to be aware of.

![chordrest_coloring_example](https://user-images.githubusercontent.com/15976214/177832876-4d8b3a95-4a52-473b-8cbb-15e1a07225be.png)

**Reply by ajyoon** [reply]
(btw, I just noticed in that screenshot the rhythm dots are not colored right, I've fixed the example script accordingly)

**Reply by LCagmac** [reply]
Thank you for sharing this!
